### ISSUE TITLE
Make Search Bar Stationary and Always Visible at Top of Page

### ISSUE THREAD

**Issue by path9263** [rtfd/sphinx_rtd_theme#758]
I use the [Read the Docs for Godot](https://docs.godotengine.org) quite a bit and find this one little thing very frustrating... The side menu bar on the left automatically scrolls down to which ever section you are currently reading about after you click on any link, this means you have to scroll all the way back up to the top of the side menu if you want to do another search.  For a project like Godot which is huge this can be a LOT of scrolling.  I would love it if the search bar could be stuck to the top of the side menu so that it is always visible.  

Thank you for this excellent open source software!

![readDocs](https://user-images.githubusercontent.com/3977110/57263119-5e048980-7024-11e9-9caf-77b91ed814ef.PNG)

**Comment by stsewd** [comment]
yeah, I think having the search form always exposed is good go have, additionally if you are using https://github.com/readthedocs/readthedocs-sphinx-search you can press `/` to open the search modal.
